### PR TITLE
Bug in "Add New Instructor" button in InstructorCourseEdit #3741

### LINKED ISSUE #3741: Bug in "Add New Instructor" button in InstructorCourseEdit
The button can be edited as shown below:

addinstructorbutton

To reproduce this bug, click and hold on the button and type words in simultaneously.

code

Adding type="button" to the input tag seems to do the trick, but it changes the button width slightly.

### PR DESCRIPTION
**Pull request by unyoungwax**
Fixes #3741

### CODE REVIEW

**Comment by wkurniawan07** [comment]
One thing: are you supposed to make a PR from your own fork or from the organization's (`nus-fboa2016-TM`) fork?

**Comment by damithc** [comment]
These individual fixes can be done from own fork.

File: src/test/resources/pages/newlyJoinedInstructorCourseEditPage.html [hunk 1]
```diff
@@ -152,7 +152,7 @@
       <br/>
       <br/>
       <div class="align-center">
-         <input class="btn btn-primary" id="btnShowNewInstructorForm" onclick="showNewInstructorForm()" value="Add New Instructor"/>
+         <input class="btn btn-primary" id="btnShowNewInstructorForm"  type="button" onclick="showNewInstructorForm()" value="Add New Instructor"/>
```
Comment (kanghj) [inline src/test/resources/pages/newlyJoinedInstructorCourseEditPage.html]: @unyoungwax, Are the changes to the html files done manually?

Comment (unyoungwax) [inline src/test/resources/pages/newlyJoinedInstructorCourseEditPage.html]: Yes

**Comment by kanghj** [comment]
Merged d6d9a

**Pull request closed without merging**